redbot/cogs/mod/kickban.py

```diff
@@ -82,6 +82,15 @@ class KickBanMixin(MixinMeta):
         elif not (0 <= days <= 7):
             return _("Invalid days. Must be between 0 and 7.")
 
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle:
+            with contextlib.suppress(discord.HTTPException):
+                em = discord.Embed(
+                    title=_(bold("You have been banned from {guild}.").format(guild=guild))
```

Comment:
I missed this in previous review.
```suggestion
                    title=bold(_("You have been banned from {guild}.").format(guild=guild))
```